examples/server_profile.tf

```diff
@@ -0,0 +1,16 @@
+provider "oneview" {
+        ov_username = "<ov_username>"
+        ov_password = "<ov_password"
+        ov_endpoint = "<ov_endpoint>"
+        ov_sslverify = false
+        ov_apiversion = <ov_apiversion>
+        ov_ifmatch = "*"
+}
+
+# Creates a server profile or Updates if already existing
```

Comment:
add the detailed example. using update, data source etc even if it is commented.

Reply:
The below code updates the server profile is it already exists in the state file. Have mentioned it in the comment. So the same code can be used for update as well.

Reply:
Please provide the data souce as well for the same.